docs(readme):  automated-lighthouse-check.com - paid / tiered (#10156)
GoogleChrome/lighthouse: readme.md

### Paid / Tiered

> * **[Automated Lighthouse Check](https://www.automated-lighthouse-check.com/)** - Lighthouse as a service offering free and premium plans. Monitoring and historical reporting of Lighthouse audits. Provides Slack, CircleCI implementation, PR comment reporting, and other "DevOps" solutions with supporting documentation.

Reviewer: ```suggestion
* **[Automated Lighthouse Check](https://www.automated-lighthouse-check.com/)** - Lighthouse-as-a-service offering free and premium plans. Provides monitoring and historical reporting of Lighthouse audits with Slack, CircleCI, GitHub, and many more integrations with supporting documentation.
```

some optional edits for you to consider :)
